symfony/psr-http-message-bridge/2.2/config/packages/psr_http_message_bridge.yaml

```diff
@@ -0,0 +1,11 @@
+psr_http_message:
+
+    # PSR-17 factories using the selected implementation (nyholm or diactoros)
+    # This functionality is enabled by default if nyholm/psr7 is available.
+    #message_factories:
+    #    enabled: true
+    #    implementation: nyholm
```

Comment:
Can we remove this option altogether and default to what is available? nyholm's by default when both are found? Maybe we can keep the option, but I would not advertise it in the recipe if we can.

Reply:
I'll have a look.

Reply:
I've removed the block from the recipe. The bundle should now enable and configure diactoros if it's the only available implementation.